### DISCUSSION TITLE
Toomanyrequests

### DISCUSSION THREAD

**Discussion by greavous1138** [louislam/dockge#488] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

So when ever I update my stacks in Dockge I get this on the second or third stack.  I can then use something like watchtower to update them fine so I'm confused why Dockge hits the pull rate limit. Couldn't find anything closed/open relating to this either by searching or manually checking the history. 
My guess is its too many too quickly as I have a few apps in each stack. Is there a way to slow down the poll rate when pulling for updates? Or should I split up my stacks and organise them another way?

edit: I forgot to mention I have also tried logging in as per the message :)

### 📝 Error Message(s) or Log

ERROR: toomanyrequests: Too Many Requests. You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limits. 

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Unraid 6.12.9

### 🌐 Browser

Chrome

### 🐋 Docker Version

Docker version 24.0.9, build 2936816

**Comment by Pootch** [comment]
I had the same problem.

That's how i solved it (TrueNAS Dockge IX-app)

Create an API token at docker hub for your account
Login at your docker host with username and api token
copy the /root/.docker/config.json to your dockge config folder ex. dockge/docker.config/
mount the the new folder to your dockge container
hostpath > your local path to config.json
mountpath: /root/.docker/config.json

restart dockge container
